Sources/FlowCommoniOSFiles/Timeline.swift

```diff
@@ -46,6 +46,10 @@ open class Timeline {
 
     public weak var delegate: TimelineDelegate?
 
+    private var resetDispatchGroup: DispatchGroup?
+    private var resetting = false
+    private var shouldAutoPlay = false
```

Comment:
maybe add comments explaining `resetting` and `shouldAutoPLay` 
could do the same for `resetDispatchGroup`, but i think that one is more self explanitory